source/stylesheets/components/input-field.scss

```diff
@@ -0,0 +1,3 @@
+.input-field-margin-bottom {
+  margin-bottom: 20px;
```

Comment:
No existing frontend style we can leverage here?

Reply:
There are the `govuk-!-padding-bottom-5` etc ones but they are in the new design system which we don't use. They may exist in the old system but I couldn't see any: https://github.com/alphagov/govuk_frontend_toolkit

Reply:
Actually Renny started using the new design system. See `_core.scss` L3. You could include the new margin classes there.